### ISSUE TITLE
Accessibility: Expansion-panel header is not marked as focused when tabbing. 

### ISSUE THREAD

**Issue by ehsantoucerkany** [angular/components#14569]
#### What is the expected behavior?
When tabbing in an expanded expansion panel, the mat-expansion-panel-header should be shaded/highlighted to indicate the focused element.  

#### What is the current behavior?
There is no way for the user to distinguish focus on the header, when tabbing to the header to collapse the already expanded item.

#### What are the steps to reproduce?
See the "Basic expansion panel example":
https://material.angular.io/components/expansion/overview

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
n/a - All versions on material.angular.io are affected: https://material.angular.io/components/expansion/overview

#### Is there anything else we should know?
The correct behaviour can be observed here: https://material-ui.com/demos/expansion-panels/

material-ui.com:
![material-ui com](https://user-images.githubusercontent.com/604690/50208041-72dfc980-0370-11e9-846a-afc792e9cc08.png)

material.angular.io:
![material angular io](https://user-images.githubusercontent.com/604690/50208049-770be700-0370-11e9-91bf-47655e30350d.png)

**Comment by literalpie** [comment]
Based on the [code](https://github.com/angular/material2/blob/master/src/lib/expansion/expansion-panel-header.scss#L14), it seems like this was an intentional decision. `background: inherit` was explicitly added to `mat-expanded:focus`

**Comment by jelbourn** [comment]
There should be _some_ focus state on the expansion header. We need to consult w/ the Material Design team to confirm what that focus treatment should be
